src/styles/header/links.scss

```diff
@@ -1,23 +1,32 @@
 $color: rgba(0, 172, 220, 1);
 
 .links {
+  position: absolute;
+  right: 50px;
+  top: 22.5px;
   display: flex;
-  justify-content: flex-end;
   width: 100%;
-  padding-right: 50px;
+  margin: 0;

```

Comment:
don't add margins (even if they are 0) to the BEM blocks. You could write it in `reset.scss` file